spec/mailers/user_mailer_spec.rb

```diff
@@ -153,4 +153,49 @@
       end
     end
   end
+  
+  describe "invite request" do
+    token = 'abc123'
+
+    before(:each) do
+      @user = FactoryGirl.create(:user)
+      @invitation = FactoryGirl.create(:invitation, token: token)
+    end
+
+    let(:email) { UserMailer.invitation(@invitation.id).deliver }
```

Comment:
I think it might be possible to skip specifying the token and creating a user, since they don't seem to be things we need to know the value of anywhere else in this test. Just this might be enough:

```
    let(:invitation) { FactoryGirl.create(:invitation) }
    let(:email) { UserMailer.invitation(invitation.id).deliver }
```